### PR TITLE
Revert "Bump Expo SDK version for exported projects (from v31 to v33)"

### PR DESCRIPTION
**Pull request by cpirich**
Reverts code-dot-org/code-dot-org#29580

SDK 33 is causing some problems with APK builds. I'd like to revert this change while I work the Expo team on this.

### CODE REVIEW

File: apps/src/templates/export/expo/App.js.ejs [hunk 1]
```diff
@@ -100,7 +100,7 @@ export default class App extends React.Component {
 
     return (
       <View onLayout={this.onLayout} style={styles.container}>
-        {!!height && indexUri && <View style={this.webViewContainerStyle()}>
+        {height && indexUri && <View style={this.webViewContainerStyle()}>
```
Comment (joshlory) [inline apps/src/templates/export/expo/App.js.ejs:103]: Unrelated change?

**Review by joshlory — APPROVED** [review]

File: apps/src/templates/export/expo/App.js.ejs (hunk 1 above)
Comment (cpirich) [reply apps/src/templates/export/expo/App.js.ejs:103]: Yes, somewhat related. I'll put that back in when I restore this work. It was a change needed when I moved to Expo SDK 33, but it could also be there with SDK 31. A simple revert of the whole thing was easier in this case.

**Comment by cpirich** [comment]
I'm going to close this without merging the revert, as I've solved the problem that required the revert with a change in this PR: https://github.com/code-dot-org/code-dot-org/pull/29686
